src/CurrencyNetwork.ts

```diff
@@ -1,75 +1,139 @@
+import * as ethUtils from 'ethereumjs-util'
+
 import { Utils } from './Utils'
 
-import * as ethUtils from 'ethereumjs-util'
+import {
+  Network,
+  NetworkDetails,
+  UserOverview,
+  UserOverviewRaw
+} from './typings'
 
+/**
+ * The CurrencyNetwork class contains all functions relevant for retrieving
+ * currency network related information.
+ */
 export class CurrencyNetwork {
+  private _utils: Utils
 
-  constructor (private utils: Utils) {
+  constructor (utils: Utils) {
+    this._utils = utils
```

Comment:
Changed it so that `typedoc` and `react-docs` can parsed it to create the **Properties** section as seen here https://trustlines-network.github.io/clientlib-docs/#tlNetwork for classes. Initially changed it only for the TLNetwork class but decided to it for all classes because of consistency.